app/controllers/finance/provider/invoices_controller.rb

```diff
@@ -12,6 +12,7 @@ class Finance::Provider::InvoicesController < Finance::Provider::BaseController
   def index
     @search = ThreeScale::Search.new(params[:search] || params)
     @invoices = collection.scope_search(@search).order_by(params[:sort], params[:direction]).paginate(paginate_params).decorate
+    @years = Invoice.years_by_provider(current_account.id).presence || [Time.zone.now.year]
```

Comment:
It is a very minor thing, but can we use `Account#timezone` for the default value?

Reply:
Yeah, no problem